vlc/rootfs/etc/cont-init.d/ingress.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Create VLC secret
+# ==============================================================================
+
+# Generate password
+if bashio::fs.file_exists /data/secret; then
+    bashio::exit.ok
+else
+    pwgen 64 1 > /data/secret
+fi
+
```

Comment:
Create own secret for the http and restore the `secret.sh` file

Reply:
Certainly can, but is there any harm in using the same secret given to discovery (Telnet) for ingress (HTTP) as well? Simplifies things somewhat, and neither are values that the user will ever need to know.

Reply:
the telnet secret is shared, it's not a lot better than the static secret, just a bit more feignedly secure. We can make an `/data/secret.ingress` and `/data/secret.integration`.

Reply:
You don't need cleanup the old secret file, that is not public jet

Reply:
I'm not sure I understand fully why it's important to use two pwgen runs for a non-user-facing secret value, but I suppose it doesn't really matter in the end.

I'll make them separate at your request.